### PR TITLE
Chapter 8: Basic Login

### PR DESCRIPTION
**Pull request by megatimranikram**

### CODE REVIEW

**Comment by megatimranikram** [comment]
ready

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -1,115 +1,19 @@
...
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+ @import 'bootstrap-sprockets';
+ @import 'bootstrap';
+
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/application.css]: delete blank line

File: app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<% provide :title, "log in" %>
+    <h1>Log in</h1>
```
Comment (ngocvt-0484) [inline app/views/sessions/new.html.erb]: format html
tab 2 space

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (megatimranikram) [reply app/assets/stylesheets/application.css]: done ms

File: app/views/sessions/new.html.erb (hunk 2 above)
Comment (megatimranikram) [reply app/views/sessions/new.html.erb]: sorry ms, what do you mean by this?
which line do i have to tab 2 spaces?

**Comment by megatimranikram** [comment]
ready

**Review by saobv-2223 — COMMENTED** [review]
Your pr is having 3 commits, merge to 1 commit please

**Comment by megatimranikram** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**